### PR TITLE
Added missing detail about TempData value for the Id of the record created after a form submission.

### PR DESCRIPTION
**Pull request by AndyButland**

### CODE REVIEW

File: Add-ons/UmbracoForms/Editor/Attaching-Workflows/index.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ If no value is selected, the message provided in **Message on submit** will be d
 
 For either method, a developer can customize the page viewed after the form is submitted on the basis of the presence of a `TempData` variable with a key of `UmbracoFormSubmitted` and a value containing the Guid identifier for the submitted form.
 
+There will also be available a similar `TempData` value containing the Guid identifier of the record that was created from the form submission.  This is found using the key `Forms_Current_Record_id`.
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Editor/Attaching-Workflows/index.md:26]: Suggestion: Try rephrasing the sentence for clarity

A similar `TempData` value is also available containing the GUID identifier of the record created from the form submission. You can find this using the `Forms_Current_Record_id` key.

**Review by eshanrnh — COMMENTED** [review]
@AndyButland Looks good. I have added a small suggestion 🙂

**Comment by AndyButland** [comment]
Much better, thanks!  Have updated.

**Comment by eshanrnh** [comment]
Merging it in. 💪

**Pull request merged**